docs/operating-scylla/procedures/cluster-management/replace-dead-node.rst

```diff
@@ -193,6 +193,14 @@ In this case, the node's data will be cleaned after restart. To remedy this, you
 
 #. Start Scylla Server
 
-   .. include:: /rst_include/scylla-commands-stop-index.rst
+   .. include:: /rst_include/scylla-commands-start-index.rst
 
 Sometimes the public/ private IP of instance is changed after restart. If so refer to the Replace Procedure_ above.
+<<<<<<< HEAD
```

Comment:
?

Reply:
This is conflict marker commited by the bot. It cannot resolve conflicts, so it just commits them and escalates to the human overlords to resolve them.